app/views/hq/sponsors/index.html.haml

```diff
@@ -1 +1,5 @@
+.action_items
+  .class
```

Comment:
I think he's referring to class action items as opposed to instance or application action items.  This naming, however awkward, is consistent with the other views